--- scim-1.4.13/configure.ac.orig	2012-04-29 00:28:56.000000000 +0200
+++ scim-1.4.13/configure.ac	2012-06-13 20:08:29.452404762 +0200
@@ -240,7 +240,7 @@
   fi
   GTK2_BINARY_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-2.0`
   GTK2_LIBDIR=`$PKG_CONFIG --variable=libdir gtk+-2.0`
-  GTK2_IM_MODULEDIR=$GTK2_LIBDIR/gtk-2.0/immodules
+  GTK2_IM_MODULEDIR=$GTK2_LIBDIR/gtk-2.0/$GTK2_BINARY_VERSION/immodules
   AC_SUBST(GTK2_IM_MODULEDIR)
   # AC_SUBST(GTK2_LIBDIR)
   # AC_SUBST(GTK2_VERSION)
@@ -251,7 +251,7 @@
   GTK3_VERSION=3.0.0
   GTK3_BINARY_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-3.0`
   GTK3_LIBDIR=`$PKG_CONFIG --variable=libdir gtk+-3.0`
-  GTK3_IM_MODULEDIR=$GTK3_LIBDIR/gtk-3.0/immodules
+  GTK3_IM_MODULEDIR=$GTK3_LIBDIR/gtk-3.0/$GTK3_BINARY_VERSION/immodules
   AC_DEFINE(HAVE_GTK_DRAW_INSERTION_CURSOR,1,[Have gtk_draw_insertion_cursor ().])
 
   AC_SUBST(GTK3_IM_MODULEDIR)
